SpatialGDKEditorToolbar/Source/Private/SpatialGDKEditorToolbarCommands.cpp

```diff
@@ -6,8 +6,8 @@
 
 void FSpatialGDKEditorToolbarCommands::RegisterCommands()
 {
-	UI_COMMAND(CreateSpatialGDKSnapshot, "Snapshot", "Creates SpatialOS Unreal GDK snapshot.", EUserInterfaceActionType::Button, FInputGesture());
 	UI_COMMAND(GenerateSchema, "Schema", "Generates schema.", EUserInterfaceActionType::Button, FInputGesture());
+	UI_COMMAND(CreateSpatialGDKSnapshot, "Snapshot", "Creates SpatialOS Unreal GDK snapshot.", EUserInterfaceActionType::Button, FInputGesture());
```

Comment:
Why did we change the order?

Reply:
This was superficial, but to match the order they appear in the toolbar.